### ISSUE TITLE
Audio stops after ~1h playback

### ISSUE THREAD

**Issue by GijsTimmers** [GijsTimmers/yamlradio#25]
Stopping gives the following error:

```
Speelt nu af: [Radio Scorpio]
Info:         [082 Georgia - Petwo, Reality Souf Broker]Traceback (most recent call last):
  File "/usr/local/bin/yamlradio", line 11, in <module>
    load_entry_point('yamlradio==2.2.1', 'console_scripts', 'yamlradio')()
  File "/usr/local/lib/python3.5/dist-packages/yamlradio/yamlradio.py", line 67, in main
    rd.stoppen()
  File "/usr/local/lib/python3.5/dist-packages/yamlradio/radio.py", line 57, in stoppen
    self.stream.stdin.flush()
BrokenPipeError: [Errno 32] Broken pipe
```

**Comment by GijsTimmers** [comment]
We need a debug mode to know what's going on.

**Comment by GijsTimmers** [comment]
Likely caused by a connection error:

```
./yamlradio-runner.py scorpio          ---INSERT---
Speelt nu af: [Radio Scorpio]
MPlayer 1.3.0 (Debian), built with gcc-5.4.0 (C) 2000-2016 MPlayer Team
mplayer: could not connect to socket
mplayer: No such file or directory
Failed to open LIRC support. You will not be able to use your remote control.

Playing http://193.190.253.57:8000/stream.
Resolving 193.190.253.57 for AF_INET6...

Couldn't resolve name for AF_INET6: 193.190.253.57
Connecting to server 193.190.253.57[193.190.253.57]: 8000...

Name   : Radio Scorpio
Genre  : Alternative
Website: http://www.radioscorpio.be
Public : yes
Bitrate: 128kbit/s
Cache size set to 320 KBytes
Cache fill:  0.00% (0 bytes)   
ICY Info: StreamTitle='M83 - Skin Of The Night';
Cache fill: 15.00% (49152 bytes)   

Audio only file format detected.
==========================================================================
Opening audio decoder: [mpg123] MPEG 1.0/2.0/2.5 layers I, II, III
AUDIO: 44100 Hz, 2 ch, s16le, 128.0 kbit/9.07% (ratio: 16000->176400)
Selected audio codec: [mpg123] afm: mpg123 (MPEG 1.0/2.0/2.5 layers I, II, III)
==========================================================================
AO: [pulse] 44100Hz 2ch s16le (2 bytes per sample)
Video: no video
Starting playback...
A:  49.7 (49.6) of -0.0 (unknown)  0.6% 0% 
Cache empty, consider increasing -cache and/or -cache-min. [performance issue]
A:  50.0 (50.0) of -0.0 (unknown) 16.0% 1% 
Audio device got stuck!
A:  50.0 (50.0) of -0.0 (unknown) 15.9% 0% 
Cache empty, consider increasing -cache and/or -cache-min. [performance issue]
A:  50.4 (50.4) of -0.0 (unknown) 55.7% 0% 


Exiting... (End of file)
Traceback (most recent call last):
  File "./yamlradio-runner.py", line 20, in <module>
    yamlradio()
  File "/home/gijs/Scripts/yamlradio/yamlradio/yamlradio.py", line 66, in main
    rd.stoppen()
  File "/home/gijs/Scripts/yamlradio/yamlradio/radio.py", line 57, in stoppen
    self.stream.stdin.write(b"q")
BrokenPipeError: [Errno 32] Broken pipe
```

**Comment by GijsTimmers** [comment]
Idea: create a thread that checks whether the subprocess is still running. If it isn't: respawn it.

**Comment by GijsTimmers** [comment]
yamlradio now uses a different mechanism to control the player. Check if this one is still valid.

**Comment by GijsTimmers** [comment]
Fixed in 8fdbabf4e63c31cceab6362d7f52da554d635874.
